scripts/import_cldr.py

```diff
@@ -880,7 +880,7 @@ def parse_interval_formats(data, tree):
             interval_formats[None] = elem.text
         elif elem.tag == "intervalFormatItem":
             skel_data = interval_formats.setdefault(elem.attrib["id"], {})
-            for item_sub in elem.getchildren():
+            for item_sub in iter(elem):
```

Comment:
```suggestion
            for item_sub in elem:
```

I don't fully understand the difference between `element.iter()` and `iter(element)` but here I believe the `iter()` call is redundant.

Reply:
Good point, I tweaked my commit accordingly.

Reply:
I still have the impression that iterating over `element.iter()` (no arguments) and iterating over `element` will iterate over the children. Hence maybe the entire code can be simplified.

Reply:
I think you are right - is #713 what you suggested?

Reply:
Yes, but I am unsure. If the code is tested, good.